PR-App/PR-App/Screens/UserMedicationDetailVC.swift

```diff
@@ -0,0 +1,148 @@
+            doseAndCapacityView.topAnchor.constraint(equalTo: medicationView.bottomAnchor, constant: 30),
+            doseAndCapacityView.leadingAnchor.constraint(equalTo: medicationView.leadingAnchor),
+            doseAndCapacityView.trailingAnchor.constraint(equalTo: medicationView.trailingAnchor),
+            doseAndCapacityView.heightAnchor.constraint(equalToConstant: 200)
+        ])
+    }
+    
+    private func configureMedicationImage() {
+        medicationView.addSubview(medicationButtonCamera)
+        
+        let settingsCellConfig = UIImage.SymbolConfiguration(pointSize: 25, weight: .regular)
+        
+        medicationButtonCamera.translatesAutoresizingMaskIntoConstraints = false
+        medicationButtonCamera.backgroundColor = .systemGray5
+        medicationButtonCamera.addTarget(self, action: #selector(imageCameraButtonTapped), for: .touchUpInside)
+        medicationButtonCamera.setImage(UIImage(systemName: "camera", withConfiguration: settingsCellConfig), for: .normal)
+        medicationButtonCamera.layer.cornerRadius = 16
+        medicationButtonCamera.tintColor = .systemGray
+        
+        NSLayoutConstraint.activate([
+            medicationButtonCamera.topAnchor.constraint(equalTo: medicationView.topAnchor),
+            medicationButtonCamera.leadingAnchor.constraint(equalTo: medicationView.leadingAnchor),
+            medicationButtonCamera.widthAnchor.constraint(equalTo: medicationView.widthAnchor, multiplier: 0.4),
+            medicationButtonCamera.bottomAnchor.constraint(equalTo: medicationView.bottomAnchor)
+        ])
+    }
+    
+    @objc private func imageCameraButtonTapped() {
+        print("button tapped")
+    }
+    
+    private func configureStackView() {
+        medicationView.addSubview(pillNameView)
+        medicationView.addSubview(packageCapacityView)
+        medicationView.addSubview(pillDoseView)
+        
+        
+        let stackView = UIStackView(arrangedSubviews: [pillNameView, packageCapacityView, pillDoseView])
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        stackView.distribution = .equalSpacing
+        medicationView.addSubview(stackView)
+        
+        NSLayoutConstraint.activate([
+            stackView.topAnchor.constraint(equalTo: medicationView.topAnchor),
+            stackView.leadingAnchor.constraint(equalTo: medicationButtonCamera.trailingAnchor, constant: 30),
+            stackView.trailingAnchor.constraint(equalTo: medicationView.trailingAnchor),
+            stackView.heightAnchor.constraint(equalTo: medicationView.heightAnchor)
+        ])
+    }
+    
+    private func configureStackView2() {
+        doseAndCapacityView.addSubview(dosageView)
+        doseAndCapacityView.addSubview(doseProgramView)
+        doseAndCapacityView.addSubview(capacityPillsLeft)
+        
+        let stackView2 = UIStackView(arrangedSubviews: [dosageView, doseProgramView, capacityPillsLeft])
```

Comment:
If You are creating view via properties, why are You creating this one locally?

Reply:
Alsoo, `stackView2` name is not the best choice, it does not tell me what this stack does.